src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -539,12 +540,102 @@ void Subgraph::createPrimitive() {
         initPluginBlockedShapes();
         initAttributes();
         initStartOffsets();
+        initBroadcastableInputs();
         optimizeIR();
     }
 
     Node::createPrimitive();
 }
 
+void Subgraph::initBroadcastableInputs() {
+    // Snippets supports tokenization of the following operations:
+    // - Unary, Binary and Ternary (Select) Elementwise ops
+    // - Softmax, MatMul, Transpose, GroupNorm
+    // Binary Elementwise ops (+ Select) requires explicit Broadcast op
+    // on inputs if broadcasting of latest dimensions is needed.
+    // These ops will be start points of DFS - need to go to Parameters and update `broadcastable_inputs_map`.
+    // We iterates through all ops by execution order. So if we already analyzied some op in the input branch - skip this branch.
+    // However, there some ops which can change `processing_dim_idx`:
+    // - Transpose has order which changes `processing_dim_idx`. But Transpose can be only before Parameters
```

Comment:
"But Transpose can be only **before** Parameters" -- after maybe?

Reply:
sure 😄 Thanks